actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/OrgBulkUploadActor.java

```diff
@@ -56,14 +58,30 @@ public void onReceive(Request request) throws Throwable {
 
   private void upload(Request request) throws IOException {
     Map<String, Object> req = (Map<String, Object>) request.getRequest().get(JsonKey.DATA);
-    validateFileHeaderFields(req, bulkOrgAllowedFields, false);
+    Map<String, Object> csvMap =
+        systemSettingClient.getSystemSettingByField(
```

Comment:
Rework not as expected.

Map<String, Object> getSystemSettingByFieldKey(getActorRef(ActorOperations.GET_SYSTEM_SETTING.getValue()), "orgProfileConfig", "csv. supportedColumns")